blob/example_test.go

```diff
@@ -169,6 +170,28 @@ func Example() {
 	// Go Cloud Development Kit
 }
 
+func ExampleBucket_ErrorAs() {
+	// This example is specific to the s3blob implementation; it demonstrates
+	// access to the underlying awserr.Error type.
+	// The types exposed for ErrorAs by s3blob are documented in
+	// https://godoc.org/gocloud.dev/blob/s3blob#hdr-As
+
+	ctx := context.Background()
+
+	b, err := blob.OpenBucket(ctx, "s3://my-bucket")
```

Comment:
It might be more clear to do the empty import for s3blob and gcsblob?

Reply:
(without those, the `OpenBucket` calls will fail).

Reply:
Done